pkg/validation/strfmt/quantity.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package strfmt
+
+import (
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type Quantity string
+
+func init() {
+	quantity := Quantity("")
+	Default.Add("quantity", &quantity, isQuantity)
```

Comment:
I wonder if this type definition / registration could be done from outside kube-openapi, e.g. defined in kubernetes/kubernetes or apiextensions-apiserver and then registered

If so, that would avoid the kube-openapi → k8s.io/apimachinery dependency